scripts/bootstrap.sh

```diff
@@ -103,16 +103,6 @@ pullDockerImages() {
     NODOCKER=$?
     if [ "${NODOCKER}" == 0 ]; then
         FABRIC_IMAGES=(peer orderer ccenv tools)
-        case "$VERSION" in
-        1.*)
-            FABRIC_IMAGES+=(javaenv)
-            shift
-            ;;
-        2.*)
-            FABRIC_IMAGES+=(nodeenv baseos javaenv)
```

Comment:
Since baseos is built from fabric repo, we can keep that one here. It is used as the basis for Go chaincode images.

Reply:
I'll add it back, but in general I just don't see the point. The tag is set by us in core.yaml, there is no harm in just letting the peer pull it, since this is the same behavior for the other two languages, it made sense to me to keep the behavior consistent.

Reply:
Added it back